doc/modules/feature_extraction.rst

```diff
@@ -916,6 +916,33 @@ Some tips and tricks:
 
     (Note that this will not filter out punctuation.)
 
+
+    The following example will, for instance, transform some British spelling 
+    to American spelling::
+
+        >>> import re
+        >>> def to_british(tokens):
+        ...     for t in tokens:
+        ...         t = re.sub(r"(...)our$", r"\1or", t)
+        ...         t = re.sub(r"([bt])re$", r"\1er", t)
+        ...         t = re.sub(r"([iy])s(e$|ing|ation)", r"\1z\2", t)
+        ...         t = re.sub(r"ogue$", "og", t)
+        ...         yield t
+        ...
+        >>> class CustomVectorizer(CountVectorizer):
+        ...     def build_tokenizer(self):
+        ...         base = super(CustomVectorizer, self).build_tokenizer()
+        ...         return lambda doc: list(to_british(base(doc)))
+        ...
+        >>> print(CustomVectorizer().build_analyzer()(u"color colour")) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        [...'color', ...'color']
+
+    for other styles of preprocessing; examples include stemming, lemmatization,
+    or normalizing numerical tokens, with the latter illustrated in:
+
+     * :ref:`sphx_glr_auto_examples_bicluster_plot_bicluster_newsgroups.py.py`
```

Comment:
Thanks, missed that. Where do you find the URL of the docs rendered by circleci? Circle CI passed, but I'm not sure if it rendered correctly..